### PR TITLE
Cocoapods

### PR DESCRIPTION
**Pull request by BernatCarbo**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -102,6 +112,13 @@ jobs:
           cd build/js/packages/doordeck-sdk
           npm publish --access public
 
+      - name: Push to CocoaPods
+        env:
+          COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
+        run: |
+          cd library/build/cocoapods/publish/release
+          pod trunk push DoordeckSDK.podspec --allow-warnings --verbose
```
Comment (RafaRuiz) [inline .github/workflows/gradle.yml:120]: should we add:
```
        pod lib lint --allow-warnings
```

for lint inspection?

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -5,6 +5,7 @@ multiplatform-settings = "1.2.0"
 nexus-publish = "2.0.0-rc-1"
 android-minSdk = "21"
 android-compileSdk = "34"
+ios-minSdk = "15"
```
Comment (mpbarnwell) [inline gradle/libs.versions.toml:8]: Can we target 13?

File: library/build.gradle.kts [hunk 3]
```diff
@@ -77,6 +78,27 @@ kotlin {
         }
     }
 
+    cocoapods {
+        summary = "Doordeck KMP SDK"
```
Comment (mpbarnwell) [inline library/build.gradle.kts:82]: Doordeck Headless SDK?

**Review by mpbarnwell — APPROVED** [review]

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (BernatCarbo) [reply .github/workflows/gradle.yml:120]: Unless I'm missing something, it doesn't seem to be necessary: https://github.com/CocoaPods/CocoaPods/issues/8098#issuecomment-422177177

File: gradle/libs.versions.toml (hunk 2 above)
Comment (BernatCarbo) [reply gradle/libs.versions.toml]: I decreased it to 14; if we can confirm that it works with 13 at any time, we can change it again

**Pull request merged**